core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -755,8 +782,14 @@ public PersistenceState getPersistenceState(long fileId) throws FileDoesNotExist
       throws AccessControlException, FileDoesNotExistException, InvalidPathException {
     Metrics.GET_FILE_INFO_OPS.inc();
     try (JournalContext journalContext = createJournalContext();
-        LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ)) {
-      mPermissionChecker.checkPermission(Mode.Bits.READ, inodePath);
+        LockedInodePath inodePath = mInodeTree.lockInodePath(path, InodeTree.LockMode.READ);
+        MasterAuditContext auditContext = createAuditContext("listStatus", path, null, null)) {
```

Comment:
I see. This information should be moved to the top of the file, since this is a required ordering for the locking and contexts of all operations. There should be similar text for the JournalContext at the top.